src/App.js

```diff
@@ -27,6 +28,7 @@ function App() {
       <div style={{marginTop: 5 +'em' /* Add spacing between navbar and page contents */}} />
       <React.Fragment>
         <Route path='/' exact component={HomePage} />
+        <Route path='/timeoff' render={(props) => <TimeOffPage {...props} clicked={setSelectedEmployee} />} />
```

Comment:
```suggestion
        <Route path='/timeoff' component={TimeOffPage} />} />
```

Your component doesn't depend on props for now (although it might change in the future depending on which direction we go). But for now it can be a standalone component.